### PR TITLE
Ignore ignored cover points

### PR DESCRIPTION
**Pull request by JairusSW**

### CODE REVIEW

**Comment by JairusSW** [comment]
Just a few small changes. Should work fine.

File: packages/glue/src/index.ts [hunk 1]
```diff
@@ -192,7 +192,8 @@ interface ICoversOptions {
  */
 export class Covers {
   private coverPoints = new Map<number, CoverPoint>();
-  // @ts-ignore
+  // Add this for early returns
+  private ignoredCoverPoints = new Array<number>()
```
Comment (jtenner) [inline packages/glue/src/index.ts]: I don't think we need to keep track of ignored cover points at all. Can't we just return early?

File: packages/glue/src/index.ts [hunk 2]
```diff
@@ -255,6 +261,8 @@ export class Covers {
    * @param id - The id of the point.
    */
   private cover(id: number): void {
+    // If it should be ignored, return.
+    if (this.ignoredCoverPoints.includes(id)) return
     // Throws if the id does not exist.
     if (!this.coverPoints.has(id))
```
Comment (jtenner) [inline packages/glue/src/index.ts]: We could just return here. I think.

File: packages/glue/src/index.ts (hunk 1 above)
Comment (JairusSW) [reply packages/glue/src/index.ts]: Okay, sure. 😀

**Comment by JairusSW** [comment]
Done.

**Pull request merged**